### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by AleksandarPetrov**

### CODE REVIEW

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ Until everything is downloaded the Portainer interface will not start.
 
 To debug this, login to the robot:
 
-    laptop $ ssh ![hostname]
+    laptop $ ssh ![linux-username]@![hostname].local
```
Comment (AndreaCensi) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md]: From the previous steps `ssh hostname` should work.

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ From another computer, you can see the contents of `/data` by visiting the URL:
     
 Otherwise, you can login via SSH and take a look at the contents of `/data`:
 
-    laptop $ ssh ![hostname].local ls /data  
+    laptop $ ssh ![linux-username]@![hostname].local ls /data
```
Comment (AndreaCensi) [inline book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md]: the previous line is wrong; it should be `ssh hostname`.

File: book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md (hunk 1 above)
Comment (AleksandarPetrov) [reply book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md]: Doesn't work for me but I might have messed up my configuration at some point.

**Comment by AleksandarPetrov** [comment]
Should be fine now.

**Pull request merged**